understanding/language-of-page.html

```diff
@@ -528,11 +462,8 @@ <h2>Key Terms</h2>
                </details>
             </section>
             <section id="test-rules">
-               <h2>Test Rules</h2>
-               <p>The following are Test Rules for certain aspects of this Success Criterion. It is
-                  not necessary to use these particular Test Rules to check for conformance with WCAG,
-                  but they are defined and approved test methods. For information on using Test Rules,
-                  see <a href="understanding-act-rules.html">Understanding Test Rules for WCAG Success Criteria</a>.
+               <h2>テストルール</h2>
+               <p>以下は、この達成基準の特定の側面に関するテストルールである。この特定のルールを使用して WCAG に適合しているかどうかを確認する必要はないが、これらのルールは定義され、承認されたテスト方法である。テストルールの使用については、<a href="understanding-act-rules.html">WCAG 達成基準のテストルールを理解する</a>を参照のこと。
                </p>
                <ul>
                   <li><a href="/WAI/standards-guidelines/act/rules/b5c3f8/">HTML page has lang attribute</a></li>
```

Comment:
テストルールのリンク先 URL が相対パスになっていたので、冒頭に「https://www.w3.org/」を追加しました。
他のテストルールのリンクも、同様に修正いただけますと幸いです。

```suggestion
                  <li><a href="https://www.w3.org/WAI/standards-guidelines/act/rules/b5c3f8/">HTML page has lang attribute</a></li>
```